pkg/management/component.go

```diff
@@ -633,6 +635,35 @@ func (c *component) SendReminderEmail(ctx context.Context, realmName string, use
 	return err
 }
 
+func (c *component) ResetSmsCounter(ctx context.Context, realmName, userID string) error {
+	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
+
+	// get the user representation
+	userKc, err := c.keycloakClient.GetUser(accessToken, realmName, userID)
+	if err != nil {
+		c.logger.Warn(ctx, "err", err.Error())
+		return err
+	}
+
+	//reset the counter, if the smsSent attribute exists
+	resetCounter := 0
+	if userKc.Attributes != nil {
+		var m = *userKc.Attributes
+		if m["smsSent"] != nil {
```

Comment:
As an option, we could avoid calling kc.UpdateUser is m["smsSent"] is nil or equal to 0 (but not an issue)

Reply:
Done